src/olympia/blocklist/mlbf.py

```diff
@@ -384,6 +398,38 @@ def should_upload_stash(
             > 0
         )
 
+    def validate(self):
+        store: Dict[str, Dict[MLBFDataType, int]] = {}
```

Comment:
nit: type annotation _within_ a function doesn't give us anything.

Reply:
It helps me keep straight what the shape of the dictionary should be. I wish python would just let us use types properly but this is the closest I could get.